tests/integrationTests/lib.py

```diff
@@ -18,6 +18,13 @@
     os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..')
 )
 
+COMPILE_CONFIGURATION_AGENT_PATH = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'compile_configure_agent.sh'
+)
+
+CONFIGURE_AGENT_PATH = os.path.join(os.getcwd(), "configure.out")
```

Comment:
Might be a good idea to pin this to a set path, e.g. `SUBMITTY_INSTALL_DIR + "/test_suite/integrationTests/configure.out"` or something?

I've increasingly grown to dislike anything that's not an "absolute" path as it leads to potential path weirdness if we ever decide to move files, or this might get run from some other location, than expected, etc. and then suddenly `os.getcwd()` is acting differently.

Reply:
Now the code looks like this:
https://github.com/Submitty/Submitty/blob/9215fa486977bbe460c45fc07dc8c4f783c0f337/tests/integrationTests/lib.py#L17-L29
Should we pin the `SUBMITTY_INSTALL_DIR` as well?

Reply:
Only way to pin `SUBMITTY_INSTALL_DIR` would be to hardcode its location to `/var/local/submitty` which while it's a pretty stable path, I'd want to avoid just on the chance we do decide to move it, or make it possible for someone to install submitty in an alternative location.

This check though ensures that we are running this from the installation directory versus from within the repository:
https://github.com/Submitty/Submitty/blob/9215fa486977bbe460c45fc07dc8c4f783c0f337/tests/integrationTests/lib.py#L41-L44